pmd/pmd.xml

```diff
@@ -56,7 +56,6 @@
 <!--      <exclude name="GuardLogStatementJavaUtil"/>-->
 <!--  </rule>-->
   <rule ref="category/java/errorprone.xml/InvalidSlf4jMessageFormat" />
-  <rule ref="category/java/errorprone.xml/LoggerIsNotStaticFinal" />
```

Comment:
https://docs.pmd-code.org/latest/pmd_release_notes_pmd7.html says to instead use `ProperLogger`